docs/settings/monitoring-settings.asciidoc

```diff
@@ -76,6 +76,9 @@ a|`monitoring.cluster_alerts.`
   health checks. By default, it matches the <<elasticsearch-pingTimeout, `elasticsearch.pingTimeout`>> setting,
   which has a default value of `30000`.
 
+| `monitoring.ui.elasticsearch.ssl`
+  | Shares the same configuration as <<elasticsearch-ssl-cert-key, `elasticsearch.ssl`>>
```

Comment:
It would be nice to clarify the use case if possible. For example, something like this:
```suggestion
  | Shares the same configuration as <<elasticsearch-ssl-cert-key, `elasticsearch.ssl`>>. These settings configure encrypted communication between Kibana and the monitoring cluster.
```